File: code/pertemuan_8/helo.php
<?php

/**
 *『Dasar pemrograman』 Function
 */

function nama_fungsi() {
    //block code yang dieksekusi
}

//function tanpa parameter
function hello() {
    echo "Hello world";
}

//function dengan parameter
function cetak_nama($nama, $kota = "Sukabumi") {
    echo "Nama saya {$nama}. Saya tinggal di kota {$kota}.";
}



//hello();
cetak_nama("Nadia", "bandung");
//cetak_nama("Nadia");

File: materi/pertemuan_8/go/Helo.go
package main
import("fmt")

func hello() {
	fmt.Println("Helo world")
}

func cetakNama(nama string, kota string) {
	fmt.Println("Nama saya " , nama, ". Saya tinggal di kota ", kota)
}

func main() {
	hello() // panggil fungsi
	cetakNama("Nadia", "Bandung")
}